### PR TITLE
Add else check to verify failure

### PR DESCRIPTION
**Pull request by savitaashture**
# Changes
Added else check to verify test case failure

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ ] [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [x] Release notes block below has been filled in or deleted (only if no user facing changes)

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by sbwsg — APPROVED** [review]

**Comment by sbwsg** [comment]
Great catch, thanks @savitaashture !

File: test/git_checkout_test.go [hunk 1]
```diff
@@ -278,6 +278,8 @@ func TestGitPipelineRunFail(t *testing.T) {
 						}
 					}
 				}
+			} else {
+				t.Fatalf("PipelineRun succeeded when should have failed")
```
Comment (pierretasci) [inline test/git_checkout_test.go:282]: There is not `format` string so you can just use `t.Fatal`

**Comment by jerop** [comment]
great catch @savitaashture! 

please add [a body to the commit with a bit of context about why we need this change](https://github.com/tektoncd/community/blob/main/standards.md#commits) :)

**Review by jerop — COMMENTED** [review]
great catch @savitaashture! 

please add [a body to the commit with a bit of context about why we need this change](https://github.com/tektoncd/community/blob/main/standards.md#commits) :)

**Comment by jerop** [comment]
fixed in https://github.com/tektoncd/pipeline/pull/4273

thanks @savitaashture

**Pull request closed without merging**